src/pf-dropdown/pf-dropdown.spec.js

```diff
@@ -24,7 +24,34 @@ describe("Patternfly Dropdown Component Test", function () {
 
     customElement.addEventListener('hidden.bs.dropdown', function () {
       expect(button.parentNode.classList.contains('open')).toBe(false);
-      done();
     }, false);
   });
+
+  it("disabled button should not open dropdown", function () {
+    var button = customElement.querySelector('.btn');
+    button.classList.add('disabled');
+    customElement.toggle();
+    expect(button.parentNode.classList.contains('open')).toBe(false);
+    button.dispatchEvent(new MouseEvent('click'));
+    expect(button.parentNode.classList.contains('open')).toBe(false);
+  });
+
+  it("should select an element", function () {
+    var buttonText = customElement.querySelector('.btn');
+    var item = customElement.querySelector('ul.dropdown-menu > li:nth-child(2)');
+    expect(buttonText.innerText).toBe('Dropdown');
+    expect(item.innerText).toBe('Item 1');
+  });
+
+  it("should not click disabled item", function () {
+    var item = customElement.querySelector('ul.dropdown-menu  li.disabled a');
+    var event = new MouseEvent('click',{
+      bubbles: true,
+      cancelable: true,
+      view: window
+    });
+    customElement.disableClick();
```

Comment:
You shouldn't call disableClick() yourself. That wouldn't be a true test for a disabled menu option.

I don't believe this is an issue with PhantomJs. Other than the disabled class on the "li" element, I don't see anything that will prevent the anchor from generating an onclick event across browsers. You may also need to disable the anchor itself. For example:

&lt;li class="disabled">&lt;a href="#" onclick="return false;">Item 3&lt/a>&lt;/li>